spec/requests/users_controller_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe 'UsersController', type: :request do
+  let!(:user) { User.create(name: 'John Doe') }
+
+  describe 'GET #index' do
+    before do
+      get users_path
+    end
+
+    context 'Applies all test to index template' do
+      it 'should check if the response status is correct' do
+        expect(response).to have_http_status(200)
+      end
+      it 'should check if a correct template was rendered' do
+        expect(response).to render_template(:index)
+      end
+      it 'should check if the response body includes the correct placeholder text' do
+        # expect(response.body).to include('Here is a list of users')
+      end
```

Comment:
- [ ] @Simpleshaikh1 Great work ensuring all the tests are passing, however, you have commented out this test, kindly uncomment it and ensure it passes went tests are run.

`Hint: I know you commented on the tests because they are not passing, this is due to you using the wrong text when testing instead of Here is a list of users it should be list of users, this is what you are using in your index page same case for the show page test.`